.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,24 +8,26 @@ assignees: ''
 ---
 
 **Describe the bug**
+
 A clear and concise description of what the bug is.
 
 **To Reproduce**
+
 Steps to reproduce the behaviour:
-0. What's your setup?
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+
+1. Provide Flux install instructions
+2. Provide a GitHub repository with Kubernetes manifests
 
 **Expected behavior**
+
 A clear and concise description of what you expected to happen.
 
 **Logs**
-If applicable, please provide logs of `fluxd` or the helm-operator. In a standard stand-alone installation of Flux, you'd get this by running `kubectl logs -n default deploy/flux`.
+
+If applicable, please provide logs of `fluxd`. In a standard stand-alone installation of Flux, you'd get this by running `kubectl logs deploy/flux  -n flux`.
 
 **Additional context**
-Add any other context about the problem here, e.g 
+
 - Flux version: 
 - Helm Operator version: 
```

Comment:
```suggestion
```

Reply:
I left this on purpose for those that report image updates issues in HelmReleases

Reply:
That's a good one

Reply:
Ack. Technically speaking image updates have little to do with the Helm operator version people are running though. The custom resource is selected based on the top level metadata and all other properties have no impact on the functionalities of Flux.